datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/init_config/jmx.yaml

```diff
@@ -12,11 +12,11 @@
     type: boolean
 - name: conf
   description: |
-    List of metrics to be collected by the integration
+    The list of metrics to be collected by the integration
     Read http://docs.datadoghq.com/integrations/java/ to learn how to customize it
     Agent 5: Customize all your metrics below
```

Comment:
Let's remove all mention of Agent versions like we did for logs since we have a separate branch for A5

Reply:
@ofek sure, done.